tmail-backend/guice/protocols/jmap/src/main/java/com/linagora/tmail/james/jmap/firebase/FirebaseModuleChooser.java

```diff
@@ -0,0 +1,25 @@
+package com.linagora.tmail.james.jmap.firebase;
+
+import java.util.List;
+
+import com.google.inject.Module;
+
+public class FirebaseModuleChooser {
+    public enum ServerType {
+        MEMORY,
+        DISTRIBUTED,
+        DISTRIBUTED_ES6
```

Comment:
Firebase integration is not opensearch/elasticsearch dependent... I think having only the option `DISTRIBUTED` is enough here :)